lib/commands/WeatherCommand.js

```diff
@@ -16,7 +16,12 @@ function getWeather(context, zip) {
     })
 
     weather.current({ method: 'zip', ZIPcode: zip, units: 'imperial' }, function(err, data) {
-        message.reply('the current weather for ' + data.name + ' is ' + data.weather[0].description + ', :thermometer: temperature is ' + data.main.temp + ' degrees. :droplet: humidity ' + data.main.humidity + '%, :wind_blowing_face: wind speed: ' + data.wind.speed + ' mph.')
+        if(typeof data.main === 'undefined') {
```

Comment:
Please add a space between `if` and the opening parentheses - this helps with consistency along the rest of the project.